### PR TITLE
fix(docker): js out and grpc out is different

### PR DESCRIPTION
**Pull request by pumano**

### CODE REVIEW

**Review by juanjoDiaz — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -31,6 +31,7 @@ RUN rm -rf /github
 
 ## Create the gRPC client
 ENV import_style=commonjs
+ENV grpc_web_import_style=commonjs
```
Comment (juanjoDiaz) [inline Dockerfile:34]: Can you revert that to the same name used by gRPC-web `grpc-web_out`?

File: README.md [hunk 2]
```diff
@@ -42,9 +42,11 @@ Defaults to `commonjs`.
 
 `import_style=commonjs`: The CommonJS style require() is also supported.
 
-`import_style=commonjs+dts`: (Experimental) In addition to above, a .d.ts typings file will also be generated for the protobuf messages and service stub.
+**Note: `commonjs+dts` and `typescript` only supported by `grpc-web_out` Use environments below:**
```
Comment (juanjoDiaz) [inline README.md]:
```
**Note: `commonjs+dts` and `typescript` are only supported by the `grpc-web_out` plugin. You should use an extra environment variable as shown bellow:**
```

File: README.md [hunk 3]
```diff
@@ -63,6 +65,12 @@ Defaults to `grpcwebtext`.
 - Payload are in the binary protobuf format.
 - Only unary calls are supported for now.
 
+## Build Docker image
+
+```bash
+docker build <path_to_Dockerfile_folder> -t juanjodiaz/grpc-web-builder
```
Comment (juanjoDiaz) [inline README.md]: Can you replace `path_to_Dockerfile_folder` by `path_to_this_repository`

**Review by juanjoDiaz — CHANGES_REQUESTED** [review]
Sorry! Few small wording changes before getting this merged 🙂

File: Dockerfile (hunk 1 above)
Comment (pumano) [reply Dockerfile:34]: looks like docker not working with environment variables that contains dash. But it works perfectly when use underscore.

Comment (pumano) [reply Dockerfile]: looks like docker not working with environment variables that contains dash. But it works perfectly when use underscore.

Comment (pumano) [reply Dockerfile]: looks like docker not working with environment variables that contains dash. But it works perfectly when use underscore.

**Review by juanjoDiaz — APPROVED** [review]

**Pull request merged**